jib-cli/src/main/java/com/google/cloud/tools/jib/cli/jar/SpringBootExplodedProcessor.java

```diff
@@ -41,20 +41,20 @@
 public class SpringBootExplodedProcessor implements JarProcessor {
 
   @Nullable private final Path jarPath;
-  @Nullable private final Path tempDirectoryPath;
+  @Nullable private final Path targetDirectoryPath;
 
-  public SpringBootExplodedProcessor(Path jarPath, Path tempDirectoryPath) {
+  public SpringBootExplodedProcessor(Path jarPath, Path targetDirectoryPath) {
     this.jarPath = jarPath;
-    this.tempDirectoryPath = tempDirectoryPath;
+    this.targetDirectoryPath = targetDirectoryPath;
   }
 
   @Override
   public List<FileEntriesLayer> createLayers() throws IOException {
-    if (tempDirectoryPath == null || jarPath == null) {
+    if (targetDirectoryPath == null || jarPath == null) {
```

Comment:
`targetDirectoryPath` can never be `null`. Do you remember why we had this null check?